api/build.gradle

```diff
@@ -69,251 +66,208 @@ apply plugin: 'org.owasp.dependencycheck'
+  __tools__RuntimeOnly.extendsFrom runtimeOnly
+
+  __swaggerCodegenV2__
+
+  generatedCompile {
+    exclude group: 'com.google.guava', module:'guava-jdk5'
+  }
+}
+
+def swagger2CodegenDir = "$buildDir/swagger2"
+
+// Configuration for the Swagger Gradle plugin:
+// https://github.com/int128/gradle-swagger-generator-plugin
+//
+// Note: The tasks below share an output directory, but we rely on the swagger
+// ignore file to ensure they don't step on eachother's outputs. If this were to
+// happen in the future due to a regression or a codegen upgrade, the issue
+// would likely manifest as an infinite build loop on our continuous incremental
+// build (e.g. dev-up). Run ./gradlew copmileGeneratedJava --info to see which
+// overlapping files/diretories may be getting written by different Swagger
+// codegen tasks.
+swaggerSources {
+  // This path is the Java sources root for finding any overridden Swagger-codegen
+  // templates we may be using.
+  def swaggerTemplateDir = "$projectDir/src/main/resources/swaggercodegen"
+  def swaggerCodegenPackage = 'org.pmiops.workbench'
+  // Raw options passed to the Swagger codegen CLI.
+  def rawOptsSwagger2 = [
+    // By default, the ignore file within the output directory is used. Override
+    // as we don't track the output directory in source.
+    '--ignore-file-override', "$projectDir/.swagger2-codegen-ignore"
+  ]
+  def baseJavaProps = [
+    'library'          : 'okhttp-gson',
+    'serializableModel': 'true',
+    'dateLibrary'      : 'java8'
+  ]
+  workbench {
+    inputFile = file('src/main/resources/workbench-api.yaml')
+    code {
+      configuration = configurations.__swaggerCodegenV2__
+      language = 'spring'
+      outputDir = file(swagger2CodegenDir)
+      templateDir = file(swaggerTemplateDir)
+      // Since the output directory is shared among these swagger tasks, wiping
+      // the directory would result in task invalidation / churn.
+      wipeOutputDir = false
+      rawOptions = rawOptsSwagger2
+      additionalProperties = [
+        'modelPackage'     : "${swaggerCodegenPackage}.model",
+        'apiPackage'       : "${swaggerCodegenPackage}.api",
+        'useTags'        : 'true',
+        // Generates delegate interfaces; used to make method annotations work without
+        // having to copy them to our implementations.
+        'delegatePattern': 'true'
+      ]
+    }
+  }
+  firecloud {
+    inputFile = file('src/main/resources/firecloud.yaml')
+    code {
+      configuration = configurations.__swaggerCodegenV2__
+      language = 'java'
+      outputDir = file(swagger2CodegenDir)
+      templateDir = file(swaggerTemplateDir)
+      wipeOutputDir = false
+      rawOptions = rawOptsSwagger2 + [
+        '--model-name-prefix', 'Firecloud'
+      ]
+      additionalProperties = baseJavaProps + [
+        'invokerPackage'   : "${swaggerCodegenPackage}.firecloud",
+        'modelPackage'     : "${swaggerCodegenPackage}.firecloud.model",
```

Comment:
nit: you could declare constants like `MODEL_SUBPACKAGE = 'model'` and `API_SUBPACKAGE` so they're not repeated for each of these. Or even have a little method for generating all the additional package properties.